README.md

```diff
@@ -155,11 +155,10 @@ image from DockerHub. The following commands require the development branch (pos
 or release 2.3.
 
 ```bash
-singularity create -s 4000 repronim.img
-singularity import repronim.img docker://repronim/simple_workflow:latest
-singularity run -B $PWD/output:/opt/repronim/simple_workflow/scripts/output -c \
+singularity build repronim.img docker://repronim/simple_workflow:latest
+singularity run -B $PWD/output:/opt/repronim/simple_workflow/scripts/output \
   --pwd /opt/repronim/simple_workflow/scripts/ repronim.img run_demo_workflow.py \
   --key 11an55u9t2TAf0EV2pHN0vOd8Ww2Gie-tHp9xGULh_dA -n 1
-singularity run -B $PWD/output:/opt/repronim/simple_workflow/scripts/output -c \
```

Comment:
is there a reason we are removing the `-c` flag?

Reply:
The `--pwd`/`--contain` interaction described in 442785e:

    The `singularity run` commands (with singularity version 2.3.2) throw
    a "No such file or directory" error because --pwd has no effect when
    it's passed with --contain (at least as of 2.3.2).  The --contain and
    --pwd issue has been reported at singularityware/singularity#1208 .
    In the meantime, drop --contain from the instructions.

Reply:
thanks for the pointer.